exercise_01/getDistanceFromThreshold.js

```diff
@@ -4,3 +4,23 @@
  * input: number (threshold)
  * output: number
  */
+
+/**
+ * Check if a number exceeds or is left to reach one value
+ * @param {number} value
+ * @param {number} threshold
+ * @returns number
+ */
+const getDistanceFromThreshold = function (value, threshold) {
+  if (value > threshold) {
+    value = value - threshold;
+    return value;
+  }
+
+  if (threshold > value) {
+    threshold = threshold - value;
+    return threshold;
+  }
+};
```

Comment:
Should'nt you add a default value for return in case "value" and "threshold" are the same?